docs/topics/api/reviewers.rst

```diff
@@ -311,16 +318,41 @@ Please note, that once a review is submitted the drafted comments are being clea
 
 .. http:get:: /api/v4/reviewers/addon/(int:addon_id)/versions/(int:version_id)/draft_comments/
 
-    Retrieve an exising draft.
+    Retrieve existing draft comments for a specific version.
 
-    :>json string comments: The comment that is being drafted as part of a review.
+    :>json string comment: The comment that is being drafted as part of a review. Specific to a line in a file.
```

Comment:
The response to this `get` request should be an array of objects, each having the fields documented here. The docs make it look like only one comment is returned from the `get` request. I guess it should be something more like `comments: [ { comment: '...', filename: '...' } ]` ?

Reply:
It is paginated - identical to the `addons` api endpoint. So that the general rules from https://addons-server.readthedocs.io/en/latest/topics/api/overview.html#pagination apply here too.